File: java/estrutura-condicional/Baskara.java
import java.util.Locale;
import java.util.Scanner;

public class Baskara {
    public static void main(String[] args) {
        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        double a, b, c, delta;
        double x, x1, x2;

        System.out.print("Coeficiente a: ");
        a = sc.nextDouble();
        System.out.print("Coeficiente b: ");
        b = sc.nextDouble();
        System.out.print("Coeficiente c: ");
        c = sc.nextDouble();

        delta = (b * b) - (4 * a * c);

        if (delta < 0) {
            System.out.println("Esta equacao nao possui raizes reais");
        } else {
            if (delta == 0) {
                x = - b / (2 * a);
                System.out.println("X = " + String.format("%.4f", x));
            } else {
                x1 = - (b - Math.sqrt(delta)) / (2 * a);
                x2 = - (b + Math.sqrt(delta)) / (2 * a);
                System.out.println("X1 = " + String.format("%.4f", x1));
                System.out.println("X2 = " + String.format("%.4f", x2));
            }
        }

        sc.close();
    }
}

File: js/estrutura-condicional/baskara.js
const readline = require("readline-sync");

const A = Number(readline.question("Coeficiente a: "));
const B = Number(readline.question("Coeficiente b: "));
const C = Number(readline.question("Coeficiente c: "));

const delta = Math.pow(B, 2) - (4 * A * C);

if (delta < 0 || delta === 0) {
    console.log("Esta equacao nao possui raizes reais");
} else {
    const X1 = - B + Math.sqrt(delta) / (2 * A);
    const X2 = - B - Math.sqrt(delta) / (2 * A);

    console.log(`X1 = ${X1.toFixed(4)}`);
    console.log(`X2 = ${X2.toFixed(4)}`);
}